x/dogfood/keeper/msg_server.go

```diff
@@ -104,5 +108,24 @@ func (k Keeper) UpdateParams(
 		nextParams.AssetIDs = prevParams.AssetIDs
 	}
 	k.SetParams(c, nextParams)
+
+	// update the related info in the AVS module
+	isAVS, avsAddr := k.avsKeeper.IsAVSByChainID(c, avstypes.ChainIDWithoutRevision(c.ChainID()))
+	if !isAVS {
+		return nil, errors.Wrapf(types.ErrNotAVSByChainID, "chainID:%s avsAddr:%s", c.ChainID(), avsAddr)
+	}
+	err := k.avsKeeper.AVSInfoUpdate(c, &avstypes.AVSRegisterOrDeregisterParams{
```

Comment:
It's better to rename this AVSInfoUpdate into UpdateAVSInfo

Reply:
fixed